includes/definitions/discovery/bintec-beip-plus.yaml

```
@@ -0,0 +1,21 @@
+# Some values are hardcoded because snmpbulkwalk is not supported
+modules:
+    os:
+        version: BIANCA-BRICK-MIB::biboAdmSWVersion.0
```

Maybe you could split this oid into version and features like so:
```suggestion
        version: BIANCA-BRICK-MIB::biboAdmSWVersion.0
        version_regex: '/^(?<version>[^ ]+) (?<features>.*)/'
```

---

Good call! I added the proper regex that works with the reported string (e.g. `Bintec Be.IP Plus V.10.2.9.102 IPv6, IPSec, PBX from 2021/03/16 00:00:00`)